### DISCUSSION TITLE
Analog JS Support

### DISCUSSION THREAD

**Discussion by manasseh-zw** [taiga-family/taiga-ui#9761] [Q&A] [answered]
Hi i recently tried to install taiga ui in an analog js project and i couldn't get it configured, it did not throw any specific errors but the components simply were not styled or working. I used ng add to install it autmoatically. Has anyone faced this challenge, please help.

**Comment by vladimirpotekhin** [comment, marked as answer]
Hi, it seems that for some reason analog-js is not picking styles from angular.json.
Try to add these styles to the global styles.css manually https://taiga-ui.dev/getting-started#styles

```
@import "@taiga-ui/core/styles/taiga-ui-theme.less";
@import "@taiga-ui/core/styles/taiga-ui-fonts.less";
```

**Reply by waterplea** [reply]
Checked with AnalogJS guys and it seems to be the expected behavior:
https://analogjs.org/docs/guides/migrating#updating-global-styles-and-scripts
